middlewares/x-powered-by/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
```

Comment:
This looks good. Could you move the bullet point under the "Removed" header?

Reply:
Just the bullet point or the entire section

Reply:
Just the bullet point.